components/fieldgroups/experience-event/experienceevent-campaign-delivery-log.example.1.json

```diff
@@ -0,0 +1,16 @@
+{
+  "@id": "0000-0000-0000-0000",
+  "xdm:timestamp": "2021-06-26T15:52:25+00:00",
+  "xdm:directMarketing": {
+    "xdm:campaignId": "100",
+    "xdm:campaignName": "JS - Offer Management",
+    "xdm:broadlogId": "102",
+    "xdm:deliveryId": "103",
+    "xdm:recipientId": "3",
+    "xdm:channel": "Email",
```

Comment:
Will have to match fixed enumeration value

Reply:
Removed enumeration from the definition and instead mapped this to the solution specific value